From 292f4c5a3a81dbeaa86dadb533ebbf02ef9ae80e Mon Sep 17 00:00:00 2001
From: Mark <alnyan@airmail.cc>
Date: Fri, 20 Dec 2019 16:23:32 +0200
Subject: [PATCH] [nyan] Scroll half the screen

---
 st.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/st.c b/st.c
index 130bf22..f47d03b 100644
--- a/st.c
+++ b/st.c
@@ -1062,7 +1062,7 @@ kscrolldown(const Arg* a)
 	int n = a->i;
 
 	if (n < 0)
-		n = term.row + n;
+		n = (term.row + n) / 2;
 
 	if (n > term.scr)
 		n = term.scr;
@@ -1080,7 +1080,7 @@ kscrollup(const Arg* a)
 	int n = a->i;
 
 	if (n < 0)
-		n = term.row + n;
+		n = (term.row + n) / 2;
 
 	if (term.scr <= HISTSIZE-n) {
 		term.scr += n;
-- 
2.23.0

